README.md

```diff
@@ -1,11 +1,35 @@
 This project was generated with [Amplication](https://amplication.com)
 
-It consists of two packages:
+It consists of three apps:
 
-### [Server](./server/README.md)
+### ecommerce-server
+Order management service for e-commerce applications.
+## Features:
```

Comment:
Please replace the order of the header and use h2 for the service name and h3 for the inner titles